packages/@lwc/engine/src/__tests__/events.spec.ts

```diff
@@ -518,13 +517,11 @@ describe('events', () => {
             div.click();
             // checking the value after the dispatch mechanism has finished
             const composedPath = event.composedPath();
-            const path = event.path;
-            // div, child's shadow, x-child, parent's shadow, x-parent, body, html, document, window
-            expect(composedPath.length).toBe(9);
-            expect(path.length).toBe(9);
+            // empty list
```

Comment:
this was a bug in synthetic shadow, native always return empty composed path after the event was completely dispatched.